.github/workflows/on-pr.yml

```diff
@@ -237,10 +237,11 @@ jobs:
     strategy:
       # The order is important for ee to be first, otherwise outputs not work correctly
       matrix:
-        name: ['novu/api-ee', 'novu/api']
+        name: ['novu/api-ee', 'novu/api', 'novu/api-ee-clerk']
```

Comment:
```suggestion
        name: ['novu/api-ee', 'novu/api', 'novu/api-ee-with-cloud-auth']
```

Reply:
When Clerk is fully released we should be able to merge it into single `novu/api-ee` tests